src/ppx/instrument.ml

```diff
@@ -890,15 +890,9 @@ struct
     in
 
     let points_data =
-      let open Parsetree in
-        let rec inline_point_offsets acc = function
-        | [] -> acc
-        | offset::more ->
-          inline_point_offsets
-            [%expr [%e Ast_builder.Default.eint ~loc offset]::[%e acc]]
-            more
-      in
-      inline_point_offsets [%expr []] points.offsets
+        Ast_builder.Default.pexp_array ~loc (List.map (
+          fun offset -> Ast_builder.Default.eint ~loc: Location.none offset
```

Comment:
The original code used `loc` for `~loc` for the elements. Is there a good reason to replace that with `Location.none`, as done here?

Reply:
No I thought the elements didn't have a loc, but was mistaken